src/scripts/main.js

```diff
@@ -1,5 +1,24 @@
 'use strict';
 
-// const pushNotification = (top, right, title, description, type) => {
-//   // write code here
-// };
+const pushNotification = (top, right, title, description, type) => {
+  const page = document.querySelector('body');
+
+  page.insertAdjacentHTML('beforeend', `
+    <div class="notification ${type}">
+      <h2 class="title">${title}</h2>
+      <p>${description}</p>
+    </div>
+  `);
+
+  const notification = document.querySelector('.notification');
+
+  notification.style.top = `${top}px`;
+  notification.style.right = `${right}px`;
+
+  setTimeout(() => {
+    notification.style.display = 'none';
+  }, 2000);
+};
```

Comment:
You can totally remove item by using
`notification.remove();`

Reply:
good idea, thanks.